internal/plugin/grpc_broker.proto

```diff
@@ -1,5 +1,5 @@
 syntax = "proto3";
-package proto;
+package plugin;
```

Comment:
I think we should be able to get away with a smaller change here where we only change this file, making the change above but also adding a `go_package` setting to override how it gets named in Go so it's still called `proto`. There is no requirement that the Go package name must match the proto package name; that's just the default.

`proto` seems like a better name for the Go package anyway, since there's already a package called `plugin`, and this would be a less invasive change.

Reply:
Well I haven't really been a maintainer in this codebase so I will cede to others who have more experience here, but the idea of having a package called `plugin` import another package called `plugin` feels more confusing to me than having a package called `proto` representing the protocol-buffers-specific stuff even though it's called "plugin" on the wire.

I tend to _expect_ wire protocols to be distinct from in-process API protocols, so the hidden dependency between them here is arguably what got us here in the first place. :man_shrugging: Even if we _do_ rename the Go package as well, I'd probably still suggest that we set the `go_package` metadata just to ensure that future changes to the Go package layout don't again inadvertently break the wire protocol.

Reply:
I would suggest that you don't call it `proto` as that will conflict with the normal import name for the protobuf package itself. E.g. `proto.Marshal` and so on.